thrall/app/lib/ElasticSearch.scala

```diff
@@ -103,6 +103,7 @@ object ElasticSearch extends ElasticSearchClient with ImageFields {
           updateLastModifiedScript,
         scriptType)
       .executeAndLog(s"updating usages on image $id")
+      .recover { case e: DocumentMissingException => {}}
```

Comment:
We could - to be descriptive of having side effects, have a return of `Future[Unit]` and then match on `UpdateResponse` to return `Unit`.

Or potentially match on `UpdateResponse`and recover on `DocumentMissingException ` to return a new type of `EsActionComplete`?